client/src/page/GamePage.js

```diff
@@ -2,16 +2,32 @@ import React from "react";
 import Game from "../game/Game";
 import styled from "@emotion/styled";
 import { PageWrapper } from "../components/PageWrapper";
+import EndPage from "./EndPage";
+import lostInSpaceImageSrc from "../assets/playerJumping.png";
+import finishedGameImageSrc from "../assets/asteronautRekate.png";
 
 const GameWrapper = styled(PageWrapper)`
   justify-content: center;
 `;
 
 function GamePage() {
   return (
-    <GameWrapper>
-      <Game />
-    </GameWrapper>
+    <>
+      {/* {!gamefinished && ( */}
+      <GameWrapper>
+        <Game />
+      </GameWrapper>
+      {/*   )}
```

Comment:
If you don't need or have the states yet, why use comments? Otherwise you could think of implementing a fake state for testing?